vendor/engines/projects/app/views/projects/shared/_select_project.html.haml

```diff
@@ -1,8 +1,7 @@
 - if order_detail.selectable_projects.any?
-  = f.input :project_id,
-    collection: order_detail.selectable_projects,
-    input_html: { class: "js--chosen", data: { placeholder: t(".placeholder") } },
-    include_blank: true
+  = f.input :project_id do
+    = f.input :project_id, collection: order_detail.selectable_projects, input_html: { class: "js--chosen hide-from-print", data: { placeholder: t(".placeholder") } }, include_blank: true
```

Comment:
Project is still behaving a little weird. There are now two "Project" labels. The input is not being hidden. I think it has to do with the `f.input :project_id` being nested inside another `f.input :project_id`.

![screen shot 2016-10-04 at 3 23 10 pm](https://cloud.githubusercontent.com/assets/1099111/19091099/c8f44d6e-8a46-11e6-9226-0743e7a3df79.png)